README.md

```diff
@@ -1,109 +1,122 @@
-
-**Note**:
-- The corresponding public key should either be shared in advance with the counterparty or retrieved by the counterparty (for example, the local scheme Certificate Authority).
-
-Because intermediary fees are not supported in the current version of the API, intermediaries involved in API message-transit may not modify the API message payload. Thus, the signature at full payload level is used to protect the integrity of the full payload of an API message from end-to-end. Regardless of how many intermediaries there are in transit, the original payload cannot be modified by the intermediaries. The final recipient of the API message must validate the signature generated by the original API client based on the message payload received.
-
-**Notes**:
-- Whether the signature needs to be validated by the intermediaries in transit is determined by the internal implementation of each intermediary or the local schema.
-- In a future version of the API, intermediary fees may be supported; at that time, signature-at-field-level may also be supported. However, both features are out-of-scope for the current version (1.0) of the API.
+[![Git Commit](https://img.shields.io/github/last-commit/mojaloop/mojaloop-specification.svg?style=flat)](https://github.com/mojaloop/mojaloop-specification/commits/master)
+[![Git Releases](https://img.shields.io/github/release/mojaloop/mojaloop-specification.svg?style=flat)](https://github.com/mojaloop/mojaloop-specification/releases)
+[![CircleCI](https://circleci.com/gh/mojaloop/mojaloop-specification.svg?style=svg)](https://circleci.com/gh/mojaloop/mojaloop-specification)
+# Open API for FSP Interoperability Specification
+The Open API for FSP Interoperability Specification includes the following documents.
+
+#### Logical Documents
+* [Logical Data Model](#logical-data-model)
+* [Generic Transaction Patterns](#generic-transaction-patterns)
+* [Use Cases](#use-cases)
+  
+#### Asynchronous REST Binding Documents
+* [API Definition](#api-definition)
+* [Central Ledger API](#central-ledger-api) 
+* [JSON Binding Rules](#json-binding-rules) 
+* [Scheme Rules](#scheme-rules)  
+  
+#### Data Integrity, Confidentiality, and Non-Repudiation
+* [PKI Best Practices](#pki-best-practices)  
+* [Signature](#signature)  
+* [Encryption](#encryption)  
+
+## Glossary
+[This document](https://github.com/mojaloop/mojaloop-specification/blob/master/fspiop-api/documents/Glossary.md) provides the glossary for the Open API (Application Programming Interface) for FSP (Financial Service Provider) Interoperability (hereafter cited as **"the API"**). Terms have been compiled from three sources:

```

Comment:
Calling every document "This document" isn't all that helpful to the reader. Maybe we can just say 'The Glossary'

Reply:
Also let's use relative links!

Reply:
I can't actually get the relative link to work when I build with gitbook locally but I will keep trying (and googling).

Reply:
I finally ended up renaming some files as spaces in filenames were causing some issues. This is a known error of gitbook.